templating/PHP.rst

```diff
@@ -196,8 +173,8 @@ decorating the template. In the ``index.html.php`` template, define a
 
 .. code-block:: html+php
 
-    <!-- templates/Hello/index.html.php -->
-    <?php $view->extend('AppBundle::layout.html.php') ?>
+    <!-- templates/hello/index.html.php -->
```

Comment:
Note that templating php engine does not supports the paths provided by Twig. See related issue https://github.com/symfony/symfony/issues/25019, So it should be here `src/Resources/views/`

Reply:
Thanks for reviewing and for reporting this error! I wasn't sure about this PHP part.